### PR TITLE
Submit some minor adjustments

### PR DESCRIPTION
**Pull request by moskytw**

### CODE REVIEW

File: content/pages/tutorial/hypothesis-testing.rst [hunk 1]
```diff
@@ -21,7 +21,7 @@ Tutorial - Hypothesis Testing in Python
 (English)
 
 This tutorial will guide attendees to conduct hypothesis testing in Python with
-real datasets, and help them use it in daily work.
+real datasets, and help attendees to use it in daily work.
```
Comment (yungyuc) [inline content/pages/tutorial/hypothesis-testing.rst]: Thanks.  Could you remove the “to” after “attendees”?  Grammatically there shouldn’t be “to” after “help”.

File: content/pages/tutorial/hypothesis-testing.rst [hunk 2]
```diff
@@ -100,19 +101,19 @@ common tests, and finally an overall guide to do a hypothesis test,
...
+2. 本工作坊會包含機率、統計學與 Python 資料處理與科學計算等內容。
    講師不會詳加介紹，但歡迎學員提問。
 
 (English)
 
 1. Attendees should understand the basic Python syntax.
-2. It will include probability, statistics, Python data processing and
-   scientific computing.  I will not introduce them in great detail, but you
-   are welcome to ask.
+2. This workshop will mention probability, statistics, Python data processing
```
Comment (yungyuc) [inline content/pages/tutorial/hypothesis-testing.rst]: I replaced all “workshop” with “tutorial” for we officially use the latter name.  Could you adjust the term here?

It’s ok to use 工作坊 in the Chinese text.

**Review by yungyuc — CHANGES_REQUESTED** [review]

**Comment by yungyuc** [comment]
Thanks for the editing.  Could you update the PR to be against `develop` branch?

File: content/pages/tutorial/hypothesis-testing.rst (hunk 1 above)
Comment (moskytw) [reply content/pages/tutorial/hypothesis-testing.rst]: Sure, but you may want to read [Help somebody (to) do – Cambridge Dictionary](https://dictionary.cambridge.org/zht/%E8%AA%9E%E6%B3%95/%E8%8B%B1%E5%BC%8F%E8%AA%9E%E6%B3%95/help-somebody-to-do).

File: content/pages/tutorial/hypothesis-testing.rst (hunk 2 above)
Comment (moskytw) [reply content/pages/tutorial/hypothesis-testing.rst]: Done.

File: content/pages/tutorial/hypothesis-testing.rst (hunk 1 above)
Comment (yungyuc) [reply content/pages/tutorial/hypothesis-testing.rst]: That's British English.  Here I am following the American English convention.

**Review by yungyuc — APPROVED** [review]

**Pull request merged**